### DISCUSSION TITLE
Animation for Wikipedia

### DISCUSSION THREAD

**Discussion by Zigoto-info** [orbiternassp/NASSP#1044] [General]
Hi ! I want to upload a 3D (POV ray) gif animation that I've made only with your FDAI texture. Can I upload it as my work ?

**Comment by Zigoto-info** [comment]
Or at least upload it ?

**Comment by indy91** [comment]
I don't understand much about this topic, but @dseagrav says:

"Wikipedia is GFDL, it shouldn't be an issue, it just has to be attributed. If you are uploading it to commons and not wikipedia they may require a license declaration, but that shouldn't be an issue. (Most media on wikipedia is via wikimedia commons)"

**Reply by Zigoto-info** [reply]
Thanks !
